src/app/modules/data-set/data-set-detail/data-set-detail.component.ts

```diff
@@ -0,0 +1,418 @@
+import { ChangeDetectorRef, Component, HostListener, OnInit, ViewChild } from '@angular/core';
+import { DataSet } from '../model/data-set/data-set.model';
+import { DataSetProject } from '../model/data-set-project/data-set-project.model';
+import { AnnotationStatus, ProjectState } from '../model/enums/enums.model';
+import { Instance } from '../model/instance/instance.model';
+import { Annotation } from '../model/annotation/annotation.model';
+import { ActivatedRoute, Params } from '@angular/router';
+import { SessionStorageService } from 'src/app/session-storage.service';
+import { MatTableDataSource } from '@angular/material/table';
+import { AddProjectDialogComponent } from '../dialogs/add-project-dialog/add-project-dialog.component';
+import { DialogConfigService } from '../dialogs/dialog-config.service';
+import { MatDialog } from '@angular/material/dialog';
+import { AnnotationConsistencyDialogComponent } from '../dialogs/annotation-consistency-dialog/annotation-consistency-dialog.component';
+import { UpdateProjectDialogComponent } from '../dialogs/update-project-dialog/update-project-dialog.component';
+import { ToastrService } from 'ngx-toastr';
+import { ConfirmDialogComponent } from '../dialogs/confirm-dialog/confirm-dialog.component';
+import { FormControl, Validators } from '@angular/forms';
+import { MatPaginator, MatPaginatorIntl } from '@angular/material/paginator';
+import { DisagreeingAnnotationsDialogComponent } from '../dialogs/disagreeing-annotations-dialog/disagreeing-annotations-dialog.component';
+import { DataSetProjectService } from '../services/data-set-project.service';
+import { DataSetService } from '../services/data-set.service';
+import { Location } from '@angular/common';
+import { AnnotationNotificationService } from '../services/annotation-notification.service';
+import { SmellCandidateInstances } from '../model/smell-candidate-instances/smell-candidate-instances.model';
+
+
+@Component({
+  selector: 'de-data-set-detail',
+  templateUrl: './data-set-detail.component.html',
+  styleUrls: ['./data-set-detail.component.css']
+})
+
+export class DataSetDetailComponent implements OnInit {
+
+  public chosenDataset: DataSet = new DataSet();
```

Comment:
Is it possible to split this into 2-3 components?

We can do a design session tomorrow to analyze this part of the project in detail.